.github/workflows/cypress-tests.yml

```
@@ -0,0 +1,51 @@
+name: E2E Tests
+
+on:
+  push:
+    branches: [patcon/236-cypress-gha]
```

This is helpful for when PR is still evolving, but we should accept this change before merging:

```suggestion
    branches:
    - dev
```